CHANGELOG.md

```diff
@@ -13,11 +13,15 @@
 - PR #500 Improved the concurrent hash map class to support partitioned (multi-pass) hash table building.
 - PR #465 Added templated C++ API for RMM to avoid explicit cast to `void**`
 
-
 ## Bug Fixes
+
 - PR #473 Added missing <random> include
 - PR #495 Updated README to correct where cffi pytest should be executed.
 
+## Non-functional changes
+
+- PR #513 `.gitignore` tweaks
```

Comment:
Let's move this to `Improvements` and not add another section to the changelog.

Reply:
@eyalroz yes, this is a code quality improvement and can fit under the `Improvements` section.

Reply:
Ok, done. Thanks for tolerating my nitpicks.